### ISSUE TITLE
3rd Bitmap Support

### ISSUE THREAD

**Issue by nikmes** [robertcs/iso4net#4]
Will it be possible to support 3rd Bitmap in this ISO library?

**Comment by robertcs** [comment]
Yes, we-re planning to support all versions as well as 3rd bitmap. I'm working on new features that hopefully will be available next month; please keep in touch
